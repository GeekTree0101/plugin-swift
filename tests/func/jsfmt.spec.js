run_spec(__dirname, ["swift"]);
