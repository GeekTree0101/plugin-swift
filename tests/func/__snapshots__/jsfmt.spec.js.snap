// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`func.swift 1`] = `
func noItems() {}

func singleItem() {
    print("bar")
}

func multipleItems() {
    print("bar")
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
func noItems() { }

func singleItem() {
    print("bar")
}

func multipleItems() {
    print("bar")
}

`;
